src/pages/mihye/Main/Main.js

```diff
@@ -0,0 +1,173 @@
+              alt="instagram_logo"
+            ></img>
+          </div>
+        </div>
+
+        <input
+          className="search_box"
+          name="search_box"
+          placeholder="검색"
+        ></input>
+
+        <ul>
+          <li className="navbar_icon_box">
+            <img
+              className="navbar_icon_compass_image"
+              src="\images\mihye\compass.png"
+              alt="compass"
+            ></img>
+          </li>
+          <li className="navbar_icon_box">
+            <img
+              className="navbar_icon_heart"
+              src="\images\mihye\heart.png"
+              alt="heart"
+            ></img>
+          </li>
+          <li className="navbar_icon_box">
+            <img
+              className="navbar_icon_reels"
+              src="\images\mihye\reels.png"
+              alt="reels"
+            ></img>
+          </li>
+        </ul>
+      </nav>
+      <div className="main">
+        <div className="feeds">
+          <article>
+            <div className="user">
+              <div className="user_profile_box">
+                <img
+                  className="user_profile_img"
+                  src="\images\mihye\user_profile_image.png"
+                  alt="user_profile_image"
+                ></img>
+                <p className="user_nickname">myeee_jeon</p>
+              </div>
+            </div>
+            <div className="feed_image">
+              <img
+                src="\images\mihye\feeds_image.png"
+                alt="user_feed_image"
+              ></img>
+            </div>
+            <div className="feed_icon_box">
+              <div className="feed_icon_left_box">
+                <img
+                  className="feed_icon_heart"
+                  src="\images\mihye\heartRed.png"
+                  alt="heratred"
+                ></img>
+                <img
+                  className="feed_icon_bubble"
+                  src="\images\mihye\bubble.png"
+                  alt="bubble"
+                ></img>
+                <img
+                  className="feed_icon_export"
+                  src="\images\mihye\export.png"
+                  alt="export"
+                ></img>
+              </div>
+              <div className="feed_icon_right_box">
+                <img
+                  className="feed_icon_bookmark"
+                  src="\images\mihye\bookmark.png"
+                  alt="bookmark"
+                ></img>
+              </div>
+            </div>
+            <div className="who_liked">jeongmin님 외 10명이 좋아합니다.</div>
+            <div className="comment_box">
+              <div className="comment">
+                {addToCommentList.map((comment, i) => {
+                  return (
+                    <>
+                      <CommentComponent
+                        comment={comment}
+                        key={i}
+                      ></CommentComponent>
+                    </>
```

Comment:
```suggestion
                      <CommentComponent
                        comment={comment}
                        key={i}
                      />
```
self-closing 할 수 있고 이미 하나의 컴포넌트만 return 하고있기 때문에 더 상위의 태그로 묶어주지 않아도 괜찮습니다